Source/WebKit/UIProcess/XR/openxr/PlatformXROpenXR.h

```diff
@@ -60,60 +60,57 @@ class OpenXRCoordinator final : public PlatformXRCoordinator {
 
 private:
     void createInstance();
-    void createSessionIfNeeded();
-    std::unique_ptr<OpenXRSwapchain> createSwapchain(uint32_t width, uint32_t height, bool alpha);
-    void cleanupSessionAndAssociatedResources();
     void initializeDevice();
     void initializeSystem();
     void initializeBlendModes();
-    void tryInitializeGraphicsBinding();
     void collectViewConfigurations();
-    bool collectSwapchainFormatsIfNeeded();
 
     struct Idle {
     };
     struct Active {
         WeakPtr<PlatformXRCoordinatorSessionEventClient> sessionEventClient;
         WebCore::PageIdentifier pageIdentifier;
         Box<RenderState> renderState;
-        RefPtr<Thread> renderThread;
+        RefPtr<WorkQueue> renderQueue;
     };
     using State = Variant<Idle, Active>;
 
-    void handleSessionStateChange(Box<RenderState>);
-    void endSessionIfExists(std::optional<WebCore::PageIdentifier>);
+    void createSessionIfNeeded();
+    void handleSessionStateChange();
+    void tryInitializeGraphicsBinding();
+    void cleanupSessionAndAssociatedResources();
+    bool collectSwapchainFormatsIfNeeded();
     enum class PollResult : bool;
-    PollResult pollEvents(Box<RenderState>);
-    PlatformXR::FrameData populateFrameData(Box<RenderState>);
+    PollResult pollEvents();
+    std::unique_ptr<OpenXRSwapchain> createSwapchain(uint32_t width, uint32_t height, bool alpha);
```

Comment:
This can be made `const` 
```suggestion
    std::unique_ptr<OpenXRSwapchain> createSwapchain(uint32_t width, uint32_t height, bool alpha) const;
```

Reply:
This is existing code, I just grouped things that are related and run in the same thread.